packages/web-components/fast-foundation/src/disclosure/disclosure.ts

```diff
@@ -0,0 +1,117 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+/**
+ * A Disclosure Custom HTML Element.
+ * Based largely on the {@link https://w3c.github.io/aria-practices/#disclosure | disclosure element }.
+ *
+ * @public
+ */
+export class Disclosure extends FASTElement {
+    /**
+     * See {@link https://www.w3.org/WAI/PF/aria/roles#button} for more information
+     * @public
+     * @remarks
+     * HTML Attribute: aria-expanded
+     */
+    @attr({ attribute: "aria-expanded" })
+    public ariaExpanded: true | false;
```

Comment:
Do we need this if it's internal? Couldn't this just be related directly to the `expanded` property?

Reply:
Not entirely sure from the A11Y aspects. Indeed `expanded` attribute is toggling in the component, and, the `aria-expanded` is given internally to the `<summary>` as well. Shall I remove it?

Reply:
`aria-expanded` is needed in the template, but it should work fine to set it's value to `expanded`. We shouldn't need both. The attribute of aria-expanded isn't really needed here as it's just storing the expanded value.

```
expanded => set aria-expanded attribute => set aria-expanded in template
```

The workflow could simply be:
```
expanded => set aria-expanded in template
```

Reply:
Yes, thanks for the explanation. Will remove it.